src/state/apis/zerion/zerionApi.ts

```diff
@@ -0,0 +1,71 @@
+import { createApi } from '@reduxjs/toolkit/dist/query/react'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { fromAssetId } from '@shapeshiftoss/caip'
+import { isEvmChainId } from '@shapeshiftoss/chain-adapters'
+import axios from 'axios'
+import { getConfig } from 'config'
+import { logger } from 'lib/logger'
+import { isSome } from 'lib/utils'
+import { BASE_RTK_CREATE_API_CONFIG } from 'state/apis/const'
+import { FEE_ASSET_IDS } from 'state/slices/selectors'
+
+import { zerionImplementationToMaybeAssetId } from './mapping'
+import { zerionFungiblesSchema } from './validators/fungible'
+
+const moduleLogger = logger.child({ module: 'zerionApi' })
+
+const ZERION_BASE_URL = 'https://api.zerion.io/v1'
+
+const options = {
+  method: 'GET' as const,
+  url: ZERION_BASE_URL,
```

Comment:
We shouldn't need this since it will be overwritten by `url` l.47.
Alternatively, in case we end up supporting more endpoints, what about making this `baseUrl` ? This way we would just have to pass relative URLs, see
https://axios-http.com/docs/req_config
https://github.com/shapeshift/web/pull/4110/files#diff-10bfda790640c77d3621ee43b8e19a3a48a926b2b411816163a86b52216d58d2R31

Reply:
nice suggestion, fixed in [`9786ba20f1727fca358bad7afc42bce5d18a6d91`](https://github.com/shapeshift/web/pull/4107/commits/cca286138d2e3cdb640ecc7dfb1a2d3fa0a47527)